ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -103,7 +105,15 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
   };
 
   // Populate series data based on query results
-  const { graphData } = useMemo(() => {
+  const { graphData } = useDeepMemo(() => {
```

Comment:
Just curious: why deep memo now instead of regular memo? Asking since that calculation is more expensive. (Although as you pointed out, definitely better than a browser crash 😂 )